examples/statistics/barchart_demo.py

```diff
@@ -88,14 +88,12 @@ def plot_student_results(student, scores, cohort_size):
     # Set the right-hand Y-axis ticks and labels
     ax2 = ax1.twinx()
 
-    # Set the tick locations
-    ax2.set_yticks(pos)
+    # Set the tick locations and labels
+    ax2.set_yticks(
+        pos, labels=[format_score(scores[k].score, k) for k in test_names])
```

Comment:
not sure if wrapping format_score in a funcformatter would be helpful here or there's already too much going on in this example

Reply:
There are some more cleanups to be done in that example. I've made a note to have a look at it later. For simplicity, I'll keep this PR as is.